server/src/config.ts

```diff
@@ -101,10 +101,20 @@ export function getConfigFromEnvironmentVariables(): {
     },
   }
 
-  const environmentVariablesUsed = Object.entries(rawConfig)
-    .map(([key, value]) => (typeof value !== 'undefined' ? key : null))
-    .filter((key): key is string => key !== null)
-    .filter((key) => key !== 'logLevel') // logLevel is a special case that we ignore
```

Comment:
Why don't we filter out `shfmt` here like we already filter out `loglevel`?

Reply:
Personal preference, if I'm honest. I trend towards exhaustivity checks, where possible, but I'm happy to switch it back to the way it was and add a test to the filter.